libs/common/src/types/tfm-deal-cancellation.ts

```diff
@@ -1,10 +1,10 @@
 import z from 'zod';
+import { ObjectId } from 'mongodb';
 import { DEAL_CANCELLATION } from '../schemas/deal-cancellation';
-import { TfmDeal } from './mongo-db-models';
 
 export type TfmDealCancellation = z.infer<typeof DEAL_CANCELLATION>;
 
 export type TfmDealCancellationResponse = {
-  cancelledDeal: TfmDeal;
+  cancelledDealUkefId: string | ObjectId;
```

Comment:
I've reverted this return type back to how it was before my previous PR
-we're only sending the data we need to tfm-api
- I've decoupled the response type from the api from the type returned in the repo layer
Happy to change this based on feedback, this feels cleanest IMO